mscore/editdrumset.cpp

```diff
@@ -111,9 +113,63 @@ EditDrumset::EditDrumset(const Drumset* ds, QWidget* parent)
       pitchList->setColumnWidth(1, 60);
       pitchList->setColumnWidth(2, 30);
 
+      QStringList validNoteheadRanges = { "Noteheads", "Slash noteheads", "Round and square noteheads" };
+      QSet<QString> excludeSym = {"noteheadParenthesisLeft", "noteheadParenthesisRight"};
+      struct SymbolIcon {
+            SymId id;
+            QIcon icon;
+            SymbolIcon(SymId i, QIcon j) : id(i), icon(j){};
+      };
+      int w = quarterCmb->iconSize().width()  * qApp->devicePixelRatio();
+      int h = quarterCmb->iconSize().height() * qApp->devicePixelRatio();
+      QList<SymbolIcon> validNoteheads;
+      for (QString range : validNoteheadRanges) {
+            for (auto symName : (*smuflRanges())[range]) {
+                   SymId id = Sym::name2id(symName);
+                   if (!excludeSym.contains(symName)) {
+                         QIcon icon;
```

Comment:
The painting here works well with some glyphs and less well with others.. no idea why for the moment. The glyph are supposed to be positioned the same.